src/auth/guard/apiKey.guard.ts

```diff
@@ -0,0 +1,5 @@
+import { Injectable } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+
+@Injectable()
+export class ApiKeyAuthGuard extends AuthGuard('x-api-key') {}
```

Comment:
Should this file be called `api-key.guard.ts` instead?

Reply:
this is updated now